draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1111,44 +1165,41 @@ document does not discuss inversion algorithms.
 Encodings are closely related to mappings.
 Like a mapping, an encoding is a function that outputs a point on an elliptic curve.
 In contrast to a mapping, however, the input to an encoding is an arbitrary-length
-byte string. Encodings can be deterministic or probabilistic.
-Deterministic encodings are preferred for security, because probabilistic
-ones are more likely to leak information through side channels. See {{VR20}}
-as an example of this type of leakage leading to a security vulnerability.
+byte string.
 
-This document constructs deterministic encodings by composing a hash function H'
+This document constructs deterministic encodings by composing a hash function Hf
 with a deterministic mapping.
-In particular, H' takes as input an arbitrary string and outputs an element of F.
+In particular, Hf takes as input an arbitrary string and outputs an element of F.
 The deterministic mapping takes that element as input and outputs a point on an
 elliptic curve E defined over F.
-Since H' takes arbitrary-length byte strings as inputs, it cannot be injective:
+Since Hf takes arbitrary-length byte strings as inputs, it cannot be injective:
 the set of inputs is larger than the set of outputs, so there must
 be distinct inputs that give the same output (i.e., there must be collisions).
-Thus, any encoding built from H' is also not injective.
+Thus, any encoding built from Hf is also not injective.
 
 Like mappings, encodings may be invertible, meaning that there is an efficient
 algorithm that, for any point P output by the encoding, outputs a string s
 such that applying the encoding to s outputs P.
-The instantiation of H' used by all encodings specified in this document ({{hashtofield}})
+The instantiation of Hf used by all encodings specified in this document ({{hashtofield}})
 is not invertible. Thus, the encodings are also not invertible.
 
+In some applications of hashing to elliptic curves, it is important that
+encodings do not leak information through side channels.
+{{VR20}} is one example of this type of leakage leading to a security vulnerability.
+{{security-considerations}} discusses further.
+
 ### Random oracle encodings {#term-rom}
 
 Two different types of encodings are possible: nonuniform encodings,
 whose output distribution of points is not uniformly random, and random
 oracle encodings, whose output distribution is indistinguishable from uniformly
-random points. Some protocols require a random oracle encoding for security, while
+random points. Some applications require a random oracle encoding for security, while
```

Comment:
I feel like this should be "protocols", given that the choice of which is made by "protocol designers" (below).